docs/hubs-user-settings.md

```diff
@@ -0,0 +1,33 @@
+---
+id: hubs-user-settings
+title: User Settings
+---
+
+## Name and Avatar
+
+The first time you enter a Hubs room you will choose a name and avatar. You can change them at any time by clicking the dropdown menu in the top left-hand corner and selecting "Set Name and Avatar." 
```

Comment:
You can also change these from the user list -> the edit pencil. Makes me wonder if we should keep it in both places.